src/main/kotlin/uk/gov/justice/digital/hmpps/externalmovementsapi/model/CreateTapAuthorisationRequest.kt

```diff
@@ -8,44 +8,59 @@ import uk.gov.justice.digital.hmpps.externalmovementsapi.entity.referencedata.Re
 import uk.gov.justice.digital.hmpps.externalmovementsapi.entity.referencedata.ReferenceDataDomain.Code.ACCOMPANIED_BY
 import uk.gov.justice.digital.hmpps.externalmovementsapi.entity.referencedata.ReferenceDataDomain.Code.LOCATION_TYPE
 import uk.gov.justice.digital.hmpps.externalmovementsapi.entity.referencedata.ReferenceDataDomain.Code.TAP_AUTHORISATION_STATUS
+import uk.gov.justice.digital.hmpps.externalmovementsapi.entity.referencedata.ReferenceDataDomain.Code.TAP_OCCURRENCE_STATUS
 import uk.gov.justice.digital.hmpps.externalmovementsapi.entity.referencedata.ReferenceDataDomain.Code.TRANSPORT
 import uk.gov.justice.digital.hmpps.externalmovementsapi.entity.referencedata.TapAuthorisationStatus
+import uk.gov.justice.digital.hmpps.externalmovementsapi.entity.referencedata.TapAuthorisationStatus.Code.APPROVED
+import uk.gov.justice.digital.hmpps.externalmovementsapi.entity.referencedata.TapAuthorisationStatus.Code.PENDING
+import uk.gov.justice.digital.hmpps.externalmovementsapi.entity.referencedata.TapOccurrenceStatus
 import java.time.LocalDate
 import java.time.LocalDateTime
 
 data class CreateTapAuthorisationRequest(
   val submittedAt: LocalDateTime,
-  val repeat: Boolean,
```

Comment:
Ah it seems logical to remove this and replace with occurrence count > 1 but Gemma confirmed that the authorisation itself is either a single or repeat. It's chosen by the user and will be passed in from the UI. They could have a repeat authorisation with 1 occurrence

Reply:
OK yes sorry, another mistake in that ticket. It's shown in the request example but I'd left it in the ACs. You did the right thing based on the ticket but it does need to go back in the request